src/content/docs/distributed-tracing/infinite-tracing/infinite-tracing-configure-data-sources.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Infinite Tracing: Configure data sources'
+tags:
+  - Understand dependencies
+  - Distributed tracing
+  - Infinite Tracing
+metaDescription: Here is why you might update the data sources of a trace observer for Infinite Tracing.
+---
+
+While some entities can be configured to send their tracing data directly to an Infinite Tracing trace observer, others can have their data routed by New Relic to a trace observer. Currently, we support this feature for Mobile, Browser, and Lambda entities. In the context of Infinite Tracing, we call these entities data sources. 
+
+## Overview
+
+There are three ways you can update the data sources on a trace observer. For each of the use cases below, you can make the change by following in instructions in our GraphQL tutorial: [Update the data sources on a trace observer](/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql#updating-the-data-sources).
+
+### ADD
+
+If you'd like to add new data sources to the existing ones on a trace observer, use the `ADD` `DataSourceGroupUpdateType`.  No data sources will be deleted with this mutation.
+
+### REPLACE
+
+If you'd like to replace your existing data sources on a trace observer with a new set of values, use the `REPLACE` `DataSourceGroupUpdateType`. Any existing values not in the new set will have their status set to `INACTIVE`.
+
+### DELETE
```

Comment:
```suggestion
### REMOVE
```

Reply:
@jonathanpdx, since I combined these docs over in #5228, here is the section that might be relevant over in that PR. Does this look correct?

```
* **REMOVE:** If you'd like to remove one or more of your existing data sources on a trace observer, use the `REMOVE` `DataSourceGroupUpdateType`. Any values passed will have their status set to `INACTIVE`.
```